### PR TITLE
Update cleanup config document

### PR DESCRIPTION
**Pull request by Sea-n**
- Add missed `units` for config definition
- Change values in two documents to the actual default value
- `syslog_purge` and `route_purge` having `order=2`, change the former to `order=1`
- Sort configs in the two documents by `order`.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Support/Cleanup-options.md [hunk 1]
```diff
@@ -14,12 +14,15 @@ These options rely on ```daily.sh``` running from cron as per the installation i
...
 ```php
-$config['syslog_purge']                              = 30;
 $config['eventlog_purge']                            = 30;
+$config['syslog_purge']                              = 30;
+$config['route_purge']                               = 10;
+$config['alert_log_purge']                           = 365;
 $config['authlog_purge']                             = 30;
+$config['ports_fdb_purge']                           = 10;
 $config['device_perf_purge']                         = 7;
-$config['rrd_purge']                                 = 90;// Not set by default
```
Comment (murrant) [inline doc/Support/Cleanup-options.md:21]: Why did you change these two to remove the notes that they aren't enabled by default?

Comment (Sea-n) [reply doc/Support/Cleanup-options.md:21]: I think the inconsistency between the default config and the document is misleading. It's clearly the units are days, not necessary to have `90` as an example. If `90` is a suggested value, I think we need to change the default config instead of the document.

For `ports_purge`, it's set to `10` instead of `true` or `false` by default. The original value and comment are misleading.

\* Although I have traced the related code, I am not 100% sure about how it works. Please let me know if something wrong/misunderstanding.

Comment (PipoCanaja) [reply doc/Support/Cleanup-options.md:21]: @Sea-n You could also add that these parameters can be configured directly and graphically in the Web GUI instead of the old way in config.php.

Comment (Sea-n) [reply doc/Support/Cleanup-options.md:21]: @PipoCanaja Thanks for the suggestion! I will add that later.

**Review by PipoCanaja — APPROVED** [review]
LGTM
